repo/object/object_reader.go

```diff
@@ -35,6 +35,10 @@ func VerifyObject(ctx context.Context, cr contentReader, oid ID) ([]content.ID,
 	return tracker.contentIDs(), nil
 }
 
+// FileReadingProgressCallback is a callback intended to be used during file copying
+// to report amount of data sent to destination.
+type FileReadingProgressCallback func(chunkSize int64)
```

Comment:
why do we need to change object reader at all, can't we just emit the progress as we're reading the bytes during restore?

Reply:
this does not seem used anywhere, the rest looks good